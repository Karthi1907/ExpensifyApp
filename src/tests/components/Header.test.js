import React from 'react';
import { shallow } from 'enzyme';

import Header from '../../components/HeaderPage';


// import ReactShallowRenderer from 'react-test-renderer/shallow';

// test('Header Page should be rendered properly', () => {
//     const renderer = new ReactShallowRenderer();
//     renderer.render(<Header />);
//     expect(renderer.getRenderOutput()).toMatchSnapshot();
// });

// import toJson from 'enzyme-to-json';

// test('Header Page should be rendered properly', () => {
//     const wrapper = shallow (<Header />);
//     expect(toJson(wrapper)).toMatchSnapshot();
// });

test('Header Page should be rendered properly', () => {
    const wrapper = shallow (<Header />);
    expect(wrapper).toMatchSnapshot();
});