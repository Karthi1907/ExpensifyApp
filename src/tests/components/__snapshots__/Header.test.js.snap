// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Page should be rendered properly 1`] = `
<div>
  <h1>
     Expensify App
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
     Dashboard 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
     Create 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
     Help 
  </NavLink>
</div>
`;
